.github/workflows/ci-update-sccache.yml

```diff
@@ -54,3 +54,40 @@ jobs:
           SCCACHE_AWS_SECRET_ACCESS_KEY: ${{ secrets.ENV_DIEM_S3_AWS_SECRET_ACCESS_KEY }}
       - uses: ./.github/actions/build-teardown
         if: ${{ steps.rust-changes.outputs.changes-found == 'true' }}
+
+  # In the future when the script can differentiate between target branches for each auto run then we'll
+  # update this report to differentiate branches.
+  unit-test-allure-report:
+    name: Unit Test Reports
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion != 'skipped' }}
+    environment:
+      name: Sccache
+    container:
+      image: diem/build_environment:main
+      volumes:
+        - "/home/runner/work/diem/diem:/opt/git/diem"
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          # This ensures that the tip of the PR is checked out instead of the merge between the base ref and the tip
+          # On `push` this value will be empty and will "do-the-right-thing"
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0 #get all the history!!!
+      - uses: ./.github/actions/build-setup
+        # build setup is called to ensure the latest docker image tooling is in place and the ~/.profile is called.
+      - name: Publish unit test results
+        uses: rexhoffman/actions/process-test-results@78f868bd1170a959f0a9864e33df725690b45b9a
```

Comment:
Point to the one in diem/actions instead?

Reply:
ahhh yeah.... meant to change that - forgot.